app/models/base.py

```diff
@@ -1,7 +1,23 @@
-from sqlalchemy.orm import DeclarativeBase
+from sqlalchemy import String, DateTime, MetaData
+from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column
+from uuid import UUID, uuid4
+from datetime import datetime, timezone
+import pytz
 
+# 한국 시간대 설정
+kst = pytz.timezone("Asia/Seoul")
 
-# Base 클래스는 SQLAlchemy의 모든 ORM 모델 클래스가 상속받는 기본 클래스입니다.
-# 이 클래스는 데이터베이스 테이블과 매핑되는 ORM 모델을 정의하는 데 사용됩니다.
+
+# 3개 다 필요함 공통된 칼럼을 상속받는 Base
 class Base(DeclarativeBase):
-    pass  # 이 클래스 자체는 비어 있지만, 상속받는 클래스들이 테이블을 정의하게 됩니다.
+
+    id: Mapped[UUID] = mapped_column(
+        UUID(as_uuid=True), primary_key=True, default=uuid4
+    )
+
+    created_at: Mapped[datetime] = mapped_column(
+        DateTime(timezone=True), default=lambda: datetime.now(kst)
+    )
+    updated_at: Mapped[datetime] = mapped_column(
+        DateTime(timezone=True), default=lambda: datetime.now(kst)
```

Comment:
```suggestion
        DateTime(timezone=True), server_default=func.now(), onupdate=func.now()
```

update_at 같은 경우도 server_default로 수정하고  onupdate=func.now() 를 넣어주시면 수정될떄 자동으로 업데이트 된 시간으로 적용되니 참고 해주시면 좋을 거 같습니다!

Reply:
넵